project/Build.scala

```diff
@@ -548,17 +547,29 @@ object Build extends sbt.Build {
       ) ++ inConfig(Test) {
         // Redefine test to run Node.js and link HelloWorld
         test := {
-          if (scalaJSStage.value == Stage.PreLink)
-            error("Can't run toolsJS/test in preLink stage")
+          val jsEnv = resolvedJSEnv.value
+          if (jsEnv.isInstanceOf[RhinoJSEnv])
+            error("Can't run toolsJS/test with Rhino")
```

Comment:
No. It relies on NodeJS internal APIs for file manipulation. Point taken. I'll reverse the test.